client/components/DatePicker/index.js

```diff
@@ -0,0 +1,13 @@
+import connectField   from 'uniforms/connectField';
+import React          from 'react';
+import { DatePicker }   from 'antd';
+
+const { RangePicker } = DatePicker;
+
+
+const DateRange = ({onChange, value: { start, stop }}) => (
+   <RangePicker value={[start, stop]} onChange={dates => onChange({start:dates[0], stop: dates[1]})} />
+);
```

Comment:
Will be great add the rest of the properties

`
const DateRange = ({onChange, value: { start, stop },...props}) => (
   <RangePicker value={[start, stop]} onChange={dates => onChange({start:dates[0], stop: dates[1]})} {... 
 filterDOMProps(props)} />
);
`

Now we can pass properties to the antd component

Reply:
@cesarve77 , I have passed the props to the antd component using filterDOMProps()